app/assets/stylesheets/application.css.sass

```diff
@@ -1,8 +1,10 @@
 @import toastr
 @import foundation_and_overrides
 @import devise_foundation_views_scss
+@import font-awesome
 @import standards/toast
 @import standards/colors
 @import standards/classes
 @import sections/header
```

Comment:
Instead of a `sections` folder, lets use the same structure of the views. `layouts/header`, `layouts/footer` and `videos/show` (it's time this action is on the `show` method instead of `index` :D).

Reply:
Alright, I'll change it.